app/controllers/api/projects.php

```diff
@@ -495,6 +497,38 @@
         $response->dynamic($project, Response::MODEL_PROJECT);
     });
 
+
+App::patch('/v1/projects/:projectId/auth/authDuration')
```

Comment:
```suggestion
App::patch('/v1/projects/:projectId/auth/duration')
```

`auth` is already being used as the namespace in the parent directory.